StanwoodCore/Protocols/HasPresenter.swift

```diff
@@ -25,8 +25,12 @@
 
 import Foundation
 
-/// :nodoc:
-/*public*/ protocol HasPresenter: AnyObject {
+/// A protocol having a *presenter* property of type Presenter.
+/// Specifically this protocol is used to define at compile time an **has** relationship between a UIViewController and a Presenter
+public protocol HasPresenter: AnyObject {
+  
     associatedtype Presenter
+  
+    /// A read/write property of type Presenter
     var presenter: Presenter! { get set }
```

Comment:
@lucaangeletti do we need this public?

Reply:
@talezion I think we can keep this property `internal`.

Reply:
@talezion Let's leave the entire protocol `public` 👍